src/components/dialogs/payment-complete.vue

```diff
@@ -55,7 +55,7 @@ import { Action, Getter } from 'vuex-class'
 import PaymentConfirm from '@/components/payment/payment-confirm.vue'
 import { NameRequestPayment } from '@/modules/payment/models'
```

Comment:
Travis, in the GH PR Files changes page, do you see a bunch of GitHub Actions warnings? Could you please clean them up while you're here?

Reply:
Yup